### PR TITLE
Add dark color theme for newsletter

### PR DESCRIPTION
**Pull request by Aloso**
Add the dark color theme again, with some minor tweaks for better readability. I checked that it works now.

### CODE REVIEW

**Comment by pickfire** [comment]
@emberian any news on this?

**Comment by nellshamrell** [comment]
I will take a look this week :) Do you mind fixing the merge conflicts?

**Comment by Aloso** [comment]
@nellshamrell the conflict is resolved, are there any other issues?

File: plugins/assets/assets.py [hunk 1]
```diff
@@ -32,7 +32,10 @@
 def add_jinja2_ext(pelican):
     """Add Webassets to Jinja2 extensions in Pelican settings."""
 
-    pelican.settings['JINJA_EXTENSIONS'].append(AssetsExtension)
+    if 'JINJA_EXTENSIONS' in pelican.settings:
+        pelican.settings['JINJA_EXTENSIONS'].append(AssetsExtension)
+    else:
+        pelican.settings['JINJA_EXTENSIONS'] = [AssetsExtension]
```
Comment (pickfire) [inline plugins/assets/assets.py:38]: Do we need these changes?

File: themes/newsletter/templates/base.html [hunk 2]
```diff
@@ -185,6 +184,51 @@
...
+       }
+
+       p {
+         color: white !important;
+       }
+
+       td .row-header-01 {
+         color: white !important;
+       }
+
```
Comment (pickfire) [inline themes/newsletter/templates/base.html:205]:
```suggestion

       td .row-header-02 {
         border-top: 1px solid #1d1d1d;
         border-bottom: 1px solid #1d1d1d;
         background-color: #040404;
       }

       .wrapper-header-01 {
         line-height: 27px;
         font-size: 23px;
         font-weight: bold;
         color: #dddddd;
       }

       .wrapper-header-02 {
         font-size: 16px;
         line-height: 36px;
         font-weight: normal;
         color: #dddddd;
       }

```

File: plugins/assets/assets.py (hunk 1 above)
Comment (Aloso) [reply plugins/assets/assets.py:38]: The previous script didn't work for me. If it worked for you previously, this change shouldn't make a difference.

Comment (pickfire) [inline plugins/assets/assets.py:38]:
```suggestion
    pelican.settings.setdefault('JINJA_EXTENSIONS', []).append(AssetsExtension)
```

**Comment by nellshamrell** [comment]
@Aloso thank you for this! I will merge it after this week's newsletter is sent out.

**Comment by nellshamrell** [comment]
Thank you for this! Merging!

**Pull request merged**